networking_cisco/plugins/cisco/device_manager/plugging_drivers/aci_vlan_trunking_driver.py

```diff
@@ -161,19 +189,32 @@ def apic_driver(self):
                     manager.NeutronManager.get_service_plugins()[
                         'GROUP_POLICY'].policy_driver_manager.policy_drivers[
                             'apic'].obj)
-                self._get_vrf_context = self._get_vrf_context_gbp
                 self._get_ext_net_name = self._get_ext_net_name_gbp
+                self._get_vrf_context = self._get_vrf_context_gbp
+                self._get_vrf_details = self._get_vrf_details_gbp
+            except KeyError:
+                    LOG.info(_("GBP service plugin not present -- skipping "
+                               "dynamic load of GBP policy driver."))
             except Exception:
-                pass
```

Comment:
how to hit this except condition? I don't see any load operation in this call:

manager.NeutronManager.get_service_plugins()[
                         'GROUP_POLICY'].policy_driver_manager.policy_drivers[
                             'apic'].obj